pom.xml

```diff
@@ -355,7 +355,7 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty</artifactId>
-        <version>3.10.5.Final</version>
+        <version>3.9.6.Final</version>
```

Comment:
I'm a little worried about this. We've had problems in the past where we have netty incompatibilities that only fail after 3 days or so of our cluster running prod. Can we try this maybe in lor1 perf?

Reply:
I checked that all of our production jars are pulling 3.9.6.Final